src/commands/compute/deploy.ts

```
@@ -196,14 +196,40 @@ export function registerComputeDeployCommand(computeCmd: Command): void {
+          if (!existing) {
+            try {
+              await ossFetch(`/api/compute/services/${encodeURIComponent(serviceId)}`, {
+                method: 'DELETE',
+              });
+              if (!json) outputInfo(`Rolled back service "${opts.name}" after build failure.`);
+            } catch {
+              if (!json)
+                outputInfo(
+                  `Build failed and rollback also failed. ` +
+                    `Run: npx @insforge/cli compute delete ${serviceId}`
+                );
```

ESLint

[error] Expected { after 'if' condition.

(curly)